src/core/sys/windows/stacktrace.d

```diff
@@ -345,26 +345,28 @@ extern(Windows) BOOL FixupDebugHeader(HANDLE hProcess, ULONG ActionCode,
     return FALSE;
 }
 
-private string generateSearchPath()
+private size_t generateSearchPath(char[] path)
```

Comment:
This size is wrong: the "search path" being generated is a concatenation of semicolon-separated paths each of which individually have that path limit.

Reply:
Then we can't really avoid GC allocations, can we ? We could however use malloc.

Reply:
I made some changes after reading your messages, it is now done in 2 pass, 1st to get the size needed, then i create the array with malloc, i don't know if that is how it should be done

Reply:
I guess a reasonable stack size would be `char[MAX_PATH * defaultPathList.length] temp`. Doing it in a single pass and only resorting to a heap allocation + 2nd pass in case that size doesn't suffice (very unlikely) would be ideal.